README.md

```
@@ -6,18 +6,15 @@ full observability.
 
 <!-- STATUS_SNIPPET_START -->
 
-> **System Status (2025-11-23) — Production Ready v0.61.3**
+> **Статус системы (n/a) — **
```

markdownlint-cli2 (0.18.1)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)